### DISCUSSION TITLE
Single project importation and exportation

### DISCUSSION THREAD

**Discussion by L-Andy** [appwrite/appwrite#3678] [Ideas]
App write handles all the basic and advanced functionalities for a fully fledged back-end, but what if we find a way of add 

1. Single project importation either in the console(better option) or an npm package
2. Single project exportation 

To the migrations easy for cases where we have a server holding multiple projects

**Comment by suleimanas** [comment]
It would be a great feature, sometimes we need to transfer the app ownership or duplicate the structure and so on

**Comment by stnguyen90** [comment]
See https://github.com/appwrite/appwrite/discussions/3043

**Reply by L-Andy** [reply]
I have checkout the cli ,but i have been getting this same issue coming up.
Every time i try running appwrite -v, the cli attempts to reinstall appwrite after it cuts the process, same with the other cli args.
![appwrite1](https://user-images.githubusercontent.com/54024696/185411540-fce97b99-0784-478f-a86e-69e4bc2ab58e.png)

![appwrite2](https://user-images.githubusercontent.com/54024696/185411634-0bc0788c-d211-457f-9cb1-07efb64adf51.png)

**Reply by suleimanas** [reply]
try to run it on separate terminal on the host not in the container

**Reply by L-Andy** [reply]
@suleimanas  i am running it from a different terminal on the host machine not in the container, but i still can't understand why it's happening like that

**Reply by stnguyen90** [reply]
@L-Andy, this doesn't look like the right CLI 👀 . Perhaps you're running the old CLI (maybe `which appwrite` will give you more info) and you need to delete it and use the new [CLI](https://github.com/appwrite/sdk-for-cli).

**Reply by L-Andy** [reply]
Yah, it was the alias in the .zshrc which was triggering the appwrite install.
Thanks great @stnguyen90
